<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="siteCombo"/>
   </item>
   <item>
    <widget class="QLineEdit" name="userEdit">
     <property name="placeholderText">
      <string>اسم المستخدم</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
     <string>كلمة المرور</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="masterEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>كلمة المرور الرئيسية</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
 </layout>
</widget>
<resources/>
<connections/>
</ui>
